src/main/resources/application-dev.yml

```diff
@@ -5,6 +5,8 @@ consentmanager:
     defaultPageSize: 20
     maxPageSize: 100
     url: http://localhost:9000
+  scheduler:
+    consentExpiryCronExpr: "0 */1 * ? * *"
```

Comment:
It's too much to run the scheduler per minute in dev environment. you can make it 1 hour.

Reply:
ok